src/overlays/actors/ovl_En_Sc_Ruppe/z_en_sc_ruppe.c

```diff
@@ -31,29 +37,155 @@ const ActorInit En_Sc_Ruppe_InitVars = {
     (ActorFunc)EnScRuppe_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BD6E40 = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_NO_PUSH | OC1_TYPE_PLAYER, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
-    { 10, 30, 0, { 0, 0, 0 } },
+RupeeInfo sRupeeInfo[] = {
+    { gameplay_keep_Tex_061FC0, 1 },   // Green rupee
+    { gameplay_keep_Tex_061FE0, 5 },   // Blue rupee
+    { gameplay_keep_Tex_062000, 20 },  // Red rupee
+    { gameplay_keep_Tex_062040, 200 }, // Orange rupee
+    { gameplay_keep_Tex_062020, 50 },  // Purple rupee
+    { gameplay_keep_Tex_062060, 10 },  // Gray rupee (unused)
```

Comment:
This is more of a note, but do we want these decimal values in an enum at all?

Reply:
this was recommended by EllipticEllipsis, i also think they provide useful information as to what kind of rupee is which.

Reply:
They are literally the rupee amounts, seems unnecessary.